packages/list/list.scss

```diff
@@ -1,40 +1,40 @@
 @import "~@fremtind/jkl-core/variables/_all.scss";
 @import "~@fremtind/jkl-core/_functions.scss";
 
-$item-space-between: $component-spacing--large;
-$item-indent: $component-spacing--xxl;
+$item-space-between: $component-spacing--small;
+$list-padding-left: $layout-spacing--small;
 
-$diamond: "\25C6";
-$diamond-hollow: "\25C7";
-$square: "\25A0";
+$black-diamond: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='100%' height='100%' viewBox='1 0 9 9' fill='none'%3E%3Cpath fill='%23000' d='M5 0l4.002 4.002L5 8.004.998 4.002z'/%3E%3C/svg%3E");
```

Comment:
Denne vises ikke i Firefox (hos meg i hvert fall)

Reply:
Firefox krever at man definerer en statisk størrelse på svg for at det skal vises reliably. Løsningen ble å sette `width` og `height` lik størrelsen på viewBox (nå 9px).